logstash-core/lib/logstash/plugins/registry.rb

```diff
@@ -123,7 +123,7 @@ def initialize(alias_registry = nil)
       @registry = java.util.concurrent.ConcurrentHashMap.new
       @java_plugins = java.util.concurrent.ConcurrentHashMap.new
       @hooks = HooksRegistry.new
-      @alias_registry = alias_registry || Java::org.logstash.plugins.AliasRegistry.new
+      @alias_registry = alias_registry || Java::org.logstash.plugins.AliasRegistry.getInstance
```

Comment:
Maybe we can use a more Ruby like syntax here, like:

```suggestion
      @alias_registry = alias_registry || Java::org.logstash.plugins.AliasRegistry.instance
```